docs/book/src/cronjob-tutorial/testdata/project/api/v1/zz_generated.deepcopy.go

```diff
@@ -52,6 +52,36 @@ func (in *CronJob) DeepCopyObject() runtime.Object {
 	return nil
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *CronJobCustomDefaulter) DeepCopyInto(out *CronJobCustomDefaulter) {
+	*out = *in
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new CronJobCustomDefaulter.
+func (in *CronJobCustomDefaulter) DeepCopy() *CronJobCustomDefaulter {
+	if in == nil {
+		return nil
+	}
+	out := new(CronJobCustomDefaulter)
+	in.DeepCopyInto(out)
+	return out
+}
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *CronJobCustomValidator) DeepCopyInto(out *CronJobCustomValidator) {
+	*out = *in
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new CronJobCustomValidator.
+func (in *CronJobCustomValidator) DeepCopy() *CronJobCustomValidator {
+	if in == nil {
+		return nil
+	}
+	out := new(CronJobCustomValidator)
+	in.DeepCopyInto(out)
+	return out
+}
+
```

Comment:
I'm not sure if we need to deepcopy the webhook custom default/validator.  This looks like it would still be on the api layer which we want to avoid so we can deal with the types away from having controller-runtime as a dependency.

Reply:
This file is from the sample used in the docs
This implementation is auto-generated by controller-gen.

Reply:
I am closing this one since `types away from having controller-runtime as a dependency.` that is not the goal of this PR